docs/buckets.rst

```diff
@@ -0,0 +1,141 @@
+Buckets
+#######
+
+.. _buckets:
+
+:ref:`Buckets <buckets>` enables the creation of collections handled by
+a group of ref:`User identifiers <user-identifiers>`.
+
+All collections are created in a bucket. By default, it uses the connected
+user's bucket (bucket only the connected user has access to).
+
+:ref:`Buckets <buckets>` can be seen as namespaces: you can have different
+collections using the same name, but stored in different buckets, so their
+names don't collide.
+
+Data (everything stored in a bucket: collections, groups and records) is
+not anymore linked to a specific user that only has access to her private data
+but is linked to the bucket and managed by bucket's owners (those who have the
+``write_bucket`` permission on the bucket.
+
+Access to buckets, groups, collections and records are granted using
+:ref:`permissions <permissions>`.
+
+To understand objects imbrication and properties, here is a little schema.
+
+.. code-block:: text
+
+               +---------------+
+               | Buckets       |
+               +---------------+
+        +----->+ - id          +<-----+
+        |      | - permissions |      |
+        |      +---------------+      |
+        |                             |
+        |                             |
+        |                             |
+        |                             |
+        |                             |
+    +---------------+        +----------------+
+    | Collections   |        | Groups         |
+    +---------------+        +----------------+
+    | - id          |        |  - id          |
+    | - permissions |        |  - members     |
+    +---------------+        |  - permissions |
+           ^                 +----------------+
+           |
+           |
+    +----------------+
+    | Records        |
+    +----------------+
+    |  - id          |
+    |  - data        |
+    |  - permissions |
+    +----------------+
+
+
+Creating a bucket
+=================
+
+A bucket can be created using a PUT on the desired bucket URI, optionally
+specifying the list of attached permissions.
+
+Arguments:
+
+- ``permissions``: A mapping object that defines the list of users for each of
+  the following permissions defined in the table below.  In any case (even if
+  not specified), the current logged-in user will get access to the bucket.
+
+Here is the list of possible permissions on a bucket:
+
++------------------------+---------------------------------+
+| Permission             | Description                     |
++========================+=================================+
+| ``write_bucket``       | The list of users principals    |
+|                        | that have administration        |
```

Comment:
administration permissions  come out of the blue here (it was mentioned as *owner* before)